templates/kafka-broker/kafka-admin-env.sh.j2

```diff
@@ -34,3 +34,8 @@ function kafka-topic-describe() {
     --describe\
     $@
 }
+
+# List partitions folder sorted by size
+function kafka-partitions() {
+  du -hs /data/disk_00/kafka_data/* | sort -hr
```

Comment:
Nope. There is some Specific here.

better do something like 

`grep "log.dirs=" /etc/kafka/server.properties | cut -d '=' -f 2 | cut -d ',' -f 1` but what if you want to parse all possible log.dirs ?

next, this file does not belong IMOO in `cp-ansbile` Il should go elsewhere where we can specifiy and add profile stuff.